modules/internal/CString.chpl

```diff
@@ -187,6 +186,13 @@ module CString {
   }
 
   //
+  // casts from c_string to c_void_ptr
+  //
+  inline proc _cast(type t, x: c_string) where t == c_void_ptr {
+    return __primitive("cast", t, x);
+  }
```

Comment:
This seems like something that could be committed independent of the ZeroMQ work.